src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerIntTest.kt

```diff
@@ -166,15 +156,6 @@ class CourtCaseEventsListenerIntTest : IntegrationTestBase() {
         },
       )
     }
-
-    await untilAsserted {
-      verify(telemetryService).trackEvent(
-        eq(TelemetryEventType.INVALID_PNC),
-        check {
-          assertThat(it["PNC]"]).isNullOrEmpty()
-        },
-      )
-    }
```

Comment:
I don't understand why this happens really but it does not seem to be desirable behaviour and the application does not actually do this when deployed as far as I can see.

perhaps it is something to do with this?
https://github.com/spring-projects/spring-data-rest/issues/1440

At all events I don't think we should assert that it happens